corehq/apps/fixtures/fixturegenerators.py

```diff
@@ -6,6 +6,22 @@
 from dimagi.utils.modules import to_function
 
 
+def item_lists_by_domain(domain):
+    ret = list()
+    for data_type in FixtureDataType.by_domain(domain):
+        ret.append({
+            'sourceUri': 'jr://fixture/item-list:%s' % data_type.tag,
+            'defaultId': data_type.tag,
+            'initialQuery': "instance('{tag}')/item-list:{tag}/{tag}".format(tag=data_type.tag),
+            'name': data_type.tag,
+            'structure': {
+                f.field_name: {
+                    'name': f.field_name,
+                    'no_option': True
```

Comment:
no_option shows up in autocomplete, but not the dropdown.

This is used if you have something like
```xml
<fixture id="item-list:other_blah" user_id="d1b70fcf93776bf2dd87267891dac6e7">
  <other_blah_list>
    <other_blah>
      <blah4>test4</blah4>
      <blah5>test5</blah5>
      <blah6>test6</blah6>
    </other_blah>
    <other_blah>
      <blah4>2test4</blah4>
      <blah5>2test5</blah5>
      <blah6>2test6</blah6>
    </other_blah>
  </other_blah_list>
</fixture>
```

From what I understood you wouldn't want drop down options of:
otehr_blah_list/other_blah/blah4
otehr_blah_list/other_blah/blah5
otehr_blah_list/other_blah/blah6

you would only want 
otehr_blah_list/other_blah

By passing no option you still get blah4 blah5 and blah6 in the autocomplete for value and label